src/constants/tokens.ts

```diff
@@ -66,13 +66,7 @@ export const PORTAL_USDC_CELO = new Token(
   'USDCet',
   'USDC (Portal from Ethereum)'
 )
-export const USDC_BASE = new Token(
-  ChainId.BASE,
-  '0xd9aAEc86B65D86f6A7B5B1b0c42FFA531710b6CA',
-  6,
-  'USD Base Coin',
-  'USDbC'
-)
+export const USDC_BASE = new Token(ChainId.BASE, '0x833589fcd6edb6e08f4c7c32d4f71b54bda02913', 6, 'USDC', 'USD Coin')
```

Comment:
USD Base Coin currently gets higher volume than native USDC on base.. do we want to keep both?

Reply:
cc @tinaszheng / @cmcewen who probably know better

Reply:
😬 id probably ask adrian! could see an argument for both.. maybe lean towards the circle native version since it's now "canonical"?

Reply:
adrian says we should keep both